terraform/modules/data-share-service/ecs.tf

```diff
@@ -0,0 +1,60 @@
+resource "aws_ecs_cluster" "main" {
+  name = var.environment
+}
+
+resource "aws_ecs_task_definition" "gdx_data_share_poc" {
+  family                   = "${var.environment}-gdx-data-share-poc"
+  requires_compatibilities = ["FARGATE"]
+  network_mode             = "awsvpc"
+  memory                   = 512
+  cpu                      = 256
```

Comment:
This seems fairly low for a JVM service, but I guess we'll have to wait and see

Reply:
Yeah I can imagine so, I just left this as a start point